internal/pkg/manifest/validate.go

```diff
@@ -593,9 +645,25 @@ func (v *vpcConfig) Validate() error {
 	if v.isEmpty() {
 		return nil
 	}
+	if err := v.Placement.Validate(); err != nil {
+		return fmt.Errorf(`validate "placement": %w`, err)
+	}
 	return nil
 }
 
+// Validate returns nil if Placement is configured correctly.
+func (p *Placement) Validate() error {
+	if p == nil {
+		return fmt.Errorf(`"placement" cannot be empty`)
+	}
+	for _, allowed := range subnetPlacements {
+		if string(*p) == allowed {
+			return nil
+		}
+	}
+	return fmt.Errorf(`"placement" %s is invalid. Must be one of %#v"`, string(*p), subnetPlacements)
```

Comment:
```suggestion
	return fmt.Errorf(`"placement" must be one of %#v"`,  subnetPlacements)
```
Samesies here. The second sentence should probably suffice.